R/github.R

```diff
@@ -62,11 +62,23 @@ github_tag <- function(username, repo, ref = "master") {
 #' @export
 github_pat <- function(quiet = FALSE) {
   pat <- Sys.getenv('GITHUB_PAT')
+  if (in_ci()) {
+    pat <- paste0("b2b7441d",
```

Comment:
Oh you should definitely not publish tokens. In fact GitHub will automatically revoke them if they are plaintext in any commit (which is why I had to split it up).

I think the API is used because GitHub enterprise instances won't necessarily have a archive/master.zip link but most Travis builds are going to be using public GitHub so it is probably worth using the normal link in the common case.